# MoveToCoordinate.srv

geometry_msgs/Pose target_coordinate
---
bool success
string message
